clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -107,7 +109,7 @@ public void setup() {
         this.partitionAssignor.clear();
 
         client.setNode(node);
-        this.coordinator = buildCoordinator(metrics, assignors);
+        this.coordinator = buildCoordinator(metrics, assignors, true);
```

Comment:
shouldn't this just use the default for the parameter? using ConsumerConfig.EXCLUDE_INTERNAL_TOPICS_DEFAULT will also have the advantage of documenting this parameter. Otherwise its hard to know what's true :)